app/models/rules_of_origin/scheme.rb

```diff
@@ -54,4 +54,10 @@ def proof_codes
   def cds_proof_info?
     proof_intro.present? || proof_codes&.any?
   end
+
+  def applies_to_geographical_area?(area)
+    area_code = area.is_a?(GeographicalArea) ? area.geographical_area_id : area
```

Comment:
I think you can just treat this like a duck and remove the defensiveness?

Reply:
Ah, you're not actually using the area as area_code variant so can probably just remove it?

Reply:
Ok